assets/themes/hailhydrant/hailhydrant.json

```diff
@@ -0,0 +1,601 @@
+{
+  "id": "HailHydrant",
+  "title": {
+    "en": "Hydrants, Extinguishers, Fire stations, and Rescue stations."
+  },
+  "shortDescription": {
+    "en": "Map to show hydrants, extinguishers, fire stations, and rescue stations."
+  },
+  "description": {
+    "en": "On this map you can find and update hydrants, fire stations,  rescue stations, and extinguishers in your favorite neighborhoods.  \n\nYou can track your precise location (mobile only) and select layers that are relevant for you in the bottom left corner. You can also use this tool to add or edit pins (points of interest) to the map and provide additional details  by answering  available questions.  \n\nAll changes you make will automatically be saved in the global database of OpenStreetMap and can be freely re-used by others."
+  },
+  "language": [
+    "en"
+  ],
+  "maintainer": "",
+  "icon": "https://wiki.openstreetmap.org/w/images/8/8c/Hailhydrant-logo.svg",
+  "version": "0",
+  "startLat": 13.67801,
+  "startLon": 121.6625,
+  "startZoom": 6,
+  "widenFactor": 0.05,
+  "socialImage": "",
+  "layers": [
+    {
+      "id": "hydrants",
+      "name": {
+        "en": "Map of hydrants"
+      },
+      "minzoom": 14,
+      "overpassTags": {
+        "and": [
+          "emergency=fire_hydrant"
+        ]
+      },
+      "title": {
+        "render": {
+          "en": "Hydrant"
+        }
+      },
+      "description": {
+        "en": "Map layer to show fire hydrants."
+      },
+      "tagRenderings": [
+        {
+          "question": {
+            "en": "What colour is the hydrant?"
+          },
+          "render": {
+            "en": "The hydrant colour is {colour}"
+          },
+          "freeform": {
+            "key": "colour"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "colour=yellow"
+                ]
+              },
+              "then": {
+                "en": "The hydrant colour is yellow"
+              }
+            }
+          ]
+        },
+        {
+          "question": {
+            "en": "What type of hydrant is it?"
+          },
+          "freeform": {
+            "key": "fire_hydrant:type"
+          },
+          "render": {
+            "en": " Hydrant type: {fire_hydrant:type}"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "fire_hydrant:type=pillar"
+                ]
+              },
+              "then": {
+                "en": "<img style=\"width:50px; height:60px\" src=\"https://upload.wikimedia.org/wikipedia/commons/thumb/f/fd/Downtown_Charlottesville_fire_hydrant_1.jpg/176px-Downtown_Charlottesville_fire_hydrant_1.jpg\" /> Pillar type."
```

Comment:
Please, include the image.

I'm also not a huge fan of having pictures to depict a type - an _icon_ is a better choice as it is more abstract. When showing a picture, some people think that the picture is of that specific hydrant, not a general picture.

Note: you might be able to steal a few icons from midgards theme here: https://mapcomplete.braindeaddev.com/fire

Reply:
Yes, I agree with you. Illustrative examples are better than photos. I'll look at what I can find there. Thanks!

Reply:
I couldn't get the icon and the text in one line.

![image](https://user-images.githubusercontent.com/5821688/114290897-840af200-9ab5-11eb-826a-75664e33b06c.png)

Any suggestions?